plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -1362,6 +1362,8 @@ fu_ccgx_hpi_device_ensure_silicon_id(FuCcgxHpiDevice *self, GError **error)
 		fu_device_add_instance_u16(FU_DEVICE(self), "SID", self->silicon_id);
 	fu_device_build_instance_id_quirk(FU_DEVICE(self), NULL, "CCGX", "SID", NULL);
 
+	g_debug("got silicon ID: %u", self->silicon_id);
```

Comment:
Not '0x%04x`? That would match the quirk IIUC?

Reply:
Argh! sure. Thanks